exporter/opentelemetry-exporter-jaeger-proto-grpc/pyproject.toml

```diff
@@ -26,6 +26,7 @@ classifiers = [
   "Typing :: Typed",
 ]
 dependencies = [
+  "protobuf ~= 3.20.0",
```

Comment:
This will cause a conflict with packages/applications trying to install newer protobuf 4. Similar to https://github.com/open-telemetry/opentelemetry-python/issues/3074

Reply:
Yes, but we are [out of support for this exporter already](https://github.com/open-telemetry/opentelemetry-python/tree/main/exporter/opentelemetry-exporter-jaeger-proto-grpc). Maybe we should consider removing it from our tests?